### ISSUE TITLE
Export : "pa_date_approbation" doit pointer vers la plus récente delib d'approbation renseignée

### ISSUE THREAD

**Issue by CeliaVermicelli** [MTES-MCT/Docurba#850]
PB : dans le csv on prend la première delib (en l'occurrence une invalide from sudocuh dans cet exemple) et non pas la dernière et plus récente. 
Lien FRP de l'exemple [Yvetot Normandie](https://docurba.beta.gouv.fr/frise/70d33da8-fa14-4dc9-8de7-5001a8017aae) (76)

<img width="967" alt="Capture d’écran 2024-10-03 à 2 23 11 PM" src="https://github.com/user-attachments/assets/946a7d54-1618-499e-9745-2a6817476953">

Que la première delibération d'approbation soit invalide ou non j'ai l'impression que le comportement attendu est de capté la date de délib la plus récente car c'est le "repère" légal. 

En lien avec la [761](https://github.com/MTES-MCT/Docurba/issues/761) ?

to do celia : 

- [x] prévenir Marie-André 76

**Comment by HermanceGauth** [comment]
@CeliaVermicelli bien résolu il me semble avec la #1115 je te passe la main

**Comment by CeliaVermicelli** [comment]
yes c'est bien la plus récente approbation qui remonte dans les exports 👌
